src/lib/utilities/conversion/units.ts

```diff
@@ -2,7 +2,9 @@ import { LanguageKeys } from '#lib/i18n/LanguageKeys';
 import { BigDecimal, div, eq, mul, pow, type Decimal } from '#lib/utilities/conversion/BigDecimal';
 import { Formulas, TemperatureUnit } from '#lib/utilities/conversion/temperature';
 import { Collection, type ReadonlyCollection } from '@discordjs/collection';
-import type { TypedT } from '@skyra/http-framework-i18n';
+import { getT, loadedLocales, type TFunction, type TypedT } from '@skyra/http-framework-i18n';
+import type { LocaleString } from 'discord-api-types/v10';
+import { toSuperscript } from './superscript.js';
```

Comment:
```suggestion
import { toSuperscript } from '#lib/utilities/conversion/superscript';
```
And let VSCode autosort this.